src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>System.Data.SqlClient.Tests</RootNamespace>
     <AssemblyName>System.Data.SqlClient.Tests</AssemblyName>
+    <KeepAllProjectReferences>true</KeepAllProjectReferences>
```

Comment:
We have added infrastructure so that we can examine a test project, find its project references, and use them to generate a new project.json which contains the dependencies from the existing project.json plus entries for the project references.  In this way, you can build your test entirely against packages instead of against the locally built product.  

That tag signals the process to ignore this project in that conversion.  It will cause the project to continue to build against project references.  This is currently required because we haven't figured out the story for publishing sni in the open, so we don't actually have a new package to restore against and that causes a failure.